rules/nette/tests/Rector/Class_/ExtractTemplateClassForPresenterRector/Source/extra_file.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Rector\Nette\Tests\Rector\Class_\ExtractTemplateClassForPresenterRector\Source;
+
+final class SomeTemplate
```

Comment:
each template should extend Nette\Bridges\ApplicationLatte\Template so we can call methods like setFile(), render() etc. on it

Reply:
I think the best way would be to extend [DefaultTemplate ](https://github.com/nette/application/blob/master/src/Bridges/ApplicationLatte/DefaultTemplate.php) but it is final